tests/php/sync/test_class.jetpack-sync-callables.php

```diff
@@ -1118,6 +1119,38 @@ function is_wpe() {
 		$this->assertEquals( $functions->get_hosting_provider_by_known_function(), 'wpe' );
 	}
 
+	/**
+	 * Test getting the main network site wpcom ID
+	 *
+	 * @return void
+	 */
+	public function test_get_main_network_site_wpcom_id() {
+		if ( ! is_multisite() ) {
+			$this->markTestSkipped( 'Only used on multisite' );
```

Comment:
Test needs to be updated to have cases for single site.

Reply:
Thanks! Added an entry to test for single site case too :) 

I'm not sure if having the test skip if it's a multisite and vice-versa is a good idea, since it generates a warning screen on the PHPUnit output.

Thoughts?

Reply:
In this case no skipping is needed as we want to the single site to run and ensure it returns the Cache SIte ID

Reply:
Updated to make the single site context test to not skip in a multi-site setup. The tests are passing locally. 🤞for Travis CI.